### ISSUE TITLE
GPAUDITLOGGING-78 custom naming of id sequence generator

### ISSUE THREAD

**Issue by robertoschwald** [robertoschwald/grails-audit-logging-plugin#89]
opened by Sophia Raw 15/Apr/15 2:25 PM

I have been using your plugin for years with out any issues, thanks!!
We are currently moving from mySql to Postgresql and in the process updating all our tables' id generators to sequences, each using it's own named sequence.
it would be fantastic if I can control the name of the sequence that gets used for the audit_log table.
Something like this in Config.groovy

```groovy
    auditLog {
      actorClosure = { request, session ->
         if (request.applicationContext.springSecurityService.principal instanceof java.lang.String)
             { return request.applicationContext.springSecurityService.principal }
             return request.applicationContext.springSecurityService.principal?.username
         }

       transactional = true        
       idMapping {
            id { generator = 'sequence' }, sequence = 'audit_log_id_seq'
       }
    }
```

**Comment by robertoschwald** [comment]
User will get full control of the AuditLogEvent class with 1.1.0. See #13

**Comment by robertoschwald** [comment]
Published 1.1.0-SNAPSHOT with externalized AuditLogEvent domain class (see #13 ) support. Would be great if you can test it.

**Comment by robertoschwald** [comment]
Use 1.1.0 for Grails2 and 2.0.0 for Grails3
